cuckoo/constants.py

```diff
@@ -48,23 +63,19 @@ class RetryConfig(TypedDict):
     retry_error_callback: NotRequired[Callable[[RetryCallState], Any]]
 
 
-class CuckooConfig(HasuraConfig):
-    retry: RetryConfig
-
-
 HASURA_HEADERS = {
     "X-Hasura-Admin-Secret": HASURA_ADMIN_SECRET,
     "X-Hasura-Role": HASURA_ROLE,
 }
-
-HASURA_DEFAULT_CONFIG: HasuraConfig = {
-    "url": HASURA_URL,
-    "headers": HASURA_HEADERS,
-}
-
 RETRY_DEFAULT_CONFIG: RetryConfig = {
     "wait": wait_random_exponential(multiplier=1, max=60),
     "stop": stop_after_attempt(5),
+    "retry": retry_if_not_exception_type(HasuraServerError),
```

Comment:
Added: When the Hasura server responds with 200 and sends an error response, we actually do not want to keep retrying as that means there is "just" an issue with the query we send and retrying will not help here. Note that this does not affect OOM errors from Hasura and such, as those raise `ConnectionError`